app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/mediadetail/view/MediaDetailFragment.kt

```diff
@@ -174,8 +174,10 @@ class MediaDetailFragment : Fragment(), Toolbar.OnMenuItemClickListener, MediaDe
                 override fun onReceive(context: Context, intent: Intent) {
                     val id = intent.getLongExtra(DownloadManager.EXTRA_DOWNLOAD_ID, -1)
                     if (downloadID == id) {
-                        Toast.makeText(context, "Download complete!\n Image " +
-                                courseUpload?.fileName + " saved to " + mDir, Toast.LENGTH_LONG).show()
+                        view?.let {
+                            Snackbar.make(it, "Download complete!\n Image " +
+                                    courseUpload?.fileName + " saved to " + mDir, Snackbar.LENGTH_LONG).show()
```

Comment:
Use existing utils method
'view?.showSnackbar(message)'
 instead of 
Snackbar.make(it1, message, Snackbar.LENGTH_LONG).show()

Add an extra parameter in ViewUtils if you need.

Reply:
alright